CHANGELOG.md

```diff
@@ -10,18 +10,24 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+> **BREAKING CHANGES**: This release has breaking changes. Please read entries
+> carefully and consult the [upgrade guide](https://grafana.com/docs/agent/latest/upgrade-guide/)
+> for specific instructions.
+
 > **DEPRECATIONS**: This release has deprecations. Please read entries
 > carefully and consult the [upgrade guide][] for specific instructions.
```

Comment:
IMO you can remove this part now, since I think noting the breaking changes takes precedence.